vcx/libvcx/build_scripts/ios/mac/mac.03.libindy.build.sh

```diff
@@ -45,7 +45,7 @@ cargo lipo --release --verbose --targets="aarch64-apple-ios,armv7-apple-ios,armv
 cd $WORK_DIR/vcx-indy-sdk/libnullpay
 
 # Replace '\"cdylib\"' with '\"staticlib\", \"cdylib\"' in Cargo.toml
-sed -i .bak 's/\"cdylib\"/\"staticlib\", \"cdylib\"/' Cargo.toml
+#sed -i .bak 's/\"cdylib\"/\"staticlib\", \"cdylib\"/' Cargo.toml
```

Comment:
Perhaps just delete this?

Reply:
The scripts for building the architectures are not mine to manage. Norm wanted this commented out because he might add it back later. Is that okay?